examples/kernel32.rs

```diff
@@ -0,0 +1,117 @@
+#![allow(non_upper_case_globals)]
```

Comment:
Since the example is using windows, we need to limit the example to windows OSes. 

```suggestion
#![cfg(windows)]
#![allow(non_upper_case_globals)]
```